.drone.star

```diff
@@ -1551,22 +1464,15 @@ def ocisServer(storage, accounts_hash_difficulty = 4):
   return [
     {
       'name': 'ocis-server',
-      'image': 'webhippie/golang:1.14',
+      'image': 'owncloudci/alpine:latest',
```

Comment:
That's "magic". I guess we can run the binary `ocis/bin/ocis` without having to install golang.